manifest.json

```diff
@@ -13,11 +13,14 @@
     "webRequestBlocking",
     "*://*.ft.com/",
     "*://*.herokuapp.com/",
+		"*://next-elastic.ft.com/*",
+		"*://search-next-elasticsearch-[a-z1-9]+.(eu-west-1|us-east-1).es.amazonaws.com/*",
     "storage"
   ],
   "background": {
     "scripts": [
-      "background.js"
+      "public/background.js",
+			"lib/hmac-sha256.js"
```

Comment:
This will need to be added to the Makefile for the `build` and `publish` commands. The lib folder wont exist at the moment.